src/boot/cic6105.c

```diff
@@ -1,3 +1,85 @@
+#include "global.h"
 #include "cic6105.h"
+#include "fault.h"
 
-u32 B_80008EE0;
+// TODO N64 fault.c functions
+void func_800AE1E0_unknown(s32, s32);
+void func_800AE258_unknown(const char*, ...);
+
+s32 func_80001714(void);
+
+extern u64 cic6105ucodeTextStart[];
+
+OSTask D_800067C0_unknown = {
+    4, 0, rspbootTextStart, 0x3E8, cic6105ucodeTextStart, 0x20, (u64*)gBuildTeam, 8, NULL, 0, NULL, 0, NULL, 0, NULL, 0,
+};
```

Comment:
I was going to suggest this be formatted like e.g. sJpegTask (and maybe add a constant for the task number) but this task number is already taken up by M_NJPEGTASK so maybe it's best to leave this til later? Guess this file will need cleanup passes later anyway

Reply:
same thoughts here

Reply:
update: I checked all the task numbers and this indeed uses the same one as JPEG, and all the ones used in GC are unchanged. So maybe we have to rename that to M_NAUXTASK or something (in the future)